tonilopezmr/day5/MazeOfJumpInstructions.kt

```diff
@@ -0,0 +1,61 @@
+package day5
+
+import org.junit.Assert.*
+import org.junit.Test
+import java.io.File
+import java.util.*
+
+class MazeOfJumpInstructions {
+
+  private fun jumpInstI(input: List<Int>): Int {
+    var offset = 0
+    var steps = 0
+    val instructions = input.toMutableList()
+
+    while (offset < input.size) {
+      val aux = instructions[offset]
+      instructions.removeAt(offset)
+      instructions.add(offset, aux + 1)
```

Comment:
I'm sure Kotlin has a `set` method for mutable lists. You can combine these 2 lines into one 👍

Reply:
I was looking for it !! thanks!

Reply:
Lol you also can do this: `instructions[offset] = aux + 1` I was trying but never success 🤔 💭  thanks